fastlane_core/spec/itunes_transporter_spec.rb

```diff
@@ -106,22 +141,36 @@ def java_download_command(provider_short_name = nil)
             expect(transporter.download('my.app.id', '/tmp')).to eq(java_download_command('abcd1234'))
           end
         end
+
+        describe "provider ID command generation" do
+          it 'generates a call to java directly' do
+            transporter = FastlaneCore::ItunesTransporter.new('fabric.devtools@gmail.com', "!> p@$s_-+=w'o%rd\"&#*<")
+            expect(transporter.provider_ids).to eq(java_provider_id_command)
+          end
+        end
       end
 
       describe "use_shell_script is true with a itc_provider short name set" do
         describe "upload command generation" do
-          it 'generates a call to java directly' do
+          it 'generates a call to the shell script' do
```

Comment:
uh, bug before?

Reply:
Yes, at least in the test description. The test condition itself wasn't bugged.